src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,75 @@
  */
 
 public class Plane {
+    private String model;
+    private String typeBySize;
+    private String typeByLanding;
+    private String typeByPurpose;
+    private String companyOwner;
 
+    public Plane(PlaneBuilder builder) {
```

Comment:
we should be able to call this constructor from inside this class only
```suggestion
   private Plane(PlaneBuilder builder) {
```